benchmarks/release_benchmarking/README.md

```diff
@@ -0,0 +1,157 @@
+
+```bash
+# Build the container
+./dev_container build \
+    --img holohub:release_benchmarking \
+    --docker_file benchmarks/release_benchmarking/Dockerfile \
+    --base_img nvcr.io/nvidia/clara-holoscan/holoscan:<holoscan-sdk-version>-$(./dev_container get_host_gpu)
+
+# Launch the dev environment
+./dev_container launch --img holohub:release_benchmarking
+
+# Inside the container, build the applications in benchmarking mode
+./run build endoscopy_tool_tracking --benchmark
+./run build multiai_ultrasound --benchmark
+
+./run build release_benchmarking
+```
+
+Run the benchmarking script with no arguments to collect performance logs in the `./output` directory.
+```bash
+./run launch release_benchmarking
+```
+
+## Summarizing Data
+
+After running benchmarks, inside the dev environment, use `run launch` to process data statistics and create bar plot PNGs: 
+```bash
+./dev_container launch --img holohub:release_benchmarking
+./run launch release_benchmarking --extra_args "--process benchmarks/release_benchmarking"
+```
+
+Alternatively, collect results across platforms. On each machine:
+1. Run benchmarks:
+```bash
+./run launch release_benchmarking
+```
+2. Add platform configuration information:
+```bash
+./run launch release_benchmarking --extra_args "--print" > benchmarks/release_benchmarking/output/platform.txt
+```
+3. Transfer output contents from each platform to a single machine:
+```bash
+# Compress information for transfer
+pushd benchmarks/release_benchmarking
+tar cvf benchmarks-<platform-name>.tar.gz output/*
+
+# Migrate the results archive with your transfer tool of choice, such as SCP
+
+# Extract results to a subfolder on the target machine
+mkdir -p output/<release>/<platform-name>/
+pushd output/<release>/<platform-name>
+tar xvf benchmarks-<platform-name>
+```
+4. Use multiple `--process` flags to generate a batch of bar plots for multiple platform results:
+```bash
+./run launch release_benchmarking --extra_args "\
+    --process benchmarks/release_benchmarking/2.4/x86_64 \
+    --process benchmarks/release_benchmarking/2.4/IGX_iGPU \
+    --process benchmarks/release/benchmarking/2.4/IGX_dGPU"
+```
+
+## Presenting Data
+
+You can use the template markdown file in the [`template`](./template/) folder to generate a markdown
+or PDF report with benchmark data with `pandoc` and `Jinja2`.
+
+1. Copy and edit `template/release.json` with information about the benchmarking configuration, including
+the release version, platform configurations, and local paths to processed data. Run
+`./run launch` to print JSON-formatted platform details to the console about the current system:
+```bash
+./dev_container launch --img holohub:release_benchmarking
+./run launch release_benchmarking --extra_args "--print"
+```
+2. Render the document with the Jinja CLI tool:
+```bash
+pushd benchmarks/release_benchmarking
+jinja2 template/results.md.tmpl template/<release-version>.json --format=json > output/<release-version>.md
```

Comment:
can we also show `pandoc` instruction?

Reply:
`pandoc` instructions are in the next section below this, marked "optional" since we currently plan to publish only the Markdown report version. OK to leave as is? Or more thoughts on presentation?